lib/child-writer/writer-proxy.js

```diff
@@ -176,11 +178,6 @@ permissions.isElevated().then((elevated) => {
       ipc.of[process.env.IPC_SERVER_ID].on('connect', () => {
         // Inherit the parent evnironment
         const childEnv = _.assign({}, process.env, {
-
-          // The CLI might call operating system utilities (like `diskutil`),
-          // so we must ensure the `PATH` is inherited.
-          PATH: process.env.PATH,
```

Comment:
Why was this line removed?

Reply:
See PR description ;)

Reply:
The PR description doesn't explain why this line was removed, but your comment above about "just pump the entire env through" does :stuck_out_tongue: